crates/partition-store/src/partition_store.rs

```diff
@@ -606,6 +609,33 @@ impl PartitionStore {
     pub fn partition(&self) -> &Arc<Partition> {
         self.db.partition()
     }
+
+    pub fn verify_deprecations(&self, key_range: RangeInclusive<PartitionKey>) {
+        // Verify invocation status v1 is unused
+        let Ok(iter) = self.iterator_from::<InvocationStatusKeyV1>(
+            TableScan::FullScanPartitionKeyRange(key_range),
```

Comment:
Did you check whether this full table scan has a noticeable effect when starting Restate with a larger db?